### DISCUSSION TITLE
Chaining various epub2tts commands

### DISCUSSION THREAD

**Discussion by mmol67** [aedocw/epub2tts-edge#27] [Q&A]
Hello. If I do somenting like
`epub2tts-edge book1.txt && epub2tts-edge book2.txt`
and first command fails, the second one executes so I end having book2.m4b and a bunch of partxx.flac, pgraphsxx.flac sntncx.mp3 files (and the inittial txt files).

 My concerns are

1. Will book2.m4b be fine? if it is using all the remaining files of the failing operation to construct the audiobook and mixing contents or not.
2.  If I then repeat `epub2tts-edge book1.txt `and no problem arises, will the book1.m4b have the right content?

Regards.

EDIT: no way, it mess things up. Maybe you can signal error when epub2tts-edge fail?

**Comment by The-Ducktor** [comment]
you could always make a batch or shell script for that kinda thing
